### ISSUE TITLE
Changing "Next" to "Upcoming version" 

### ISSUE THREAD

**Issue by spzala** [etcd-io/website#220]
The "Next" under https://etcd.io/docs/ seems missing something - can we change it to "Upcoming" or "Upcoming version" or "Current development" something like that. 
Also, once clicking Next, under https://etcd.io/docs/next/ it shows "Next Doc" which is also little difficult to read. 

Opening this issue for brainstorming.  

@nate-double-u @chalin @ptabor @gyuho

**Comment by chalin** [comment]
I agree. I don't find "next" suitable, or very communicative of what the docs are -- but it seemed better than the previous name, "current".

I think that we should be explicit say which version these are a draft of, e.g.: **v3.5.0 DRAFT Docs** (or whatever the next release will be).

Thoughts?

**Comment by chalin** [comment]
Just for context, the suggestion for "next" was inspired, in part from sites like: https://next.angular.io/docs (vs. the latest stable version being at https://angular.io/docs) -- angular also has "rc" docs for release candidates (not something I believe we need for etcd).

**Comment by chalin** [comment]
- [ ] Propagate this change to the `docsy` branch

**Comment by chalin** [comment]
All: https://github.com/etcd-io/website/pull/226 is a proposed fix for this. PTAL

**Comment by chalin** [comment]
@spzala et al: on the docsy branch, #237 in addition proposes the addition of a warning banner at the top of the "next" doc pages. For example:

> ![image](https://user-images.githubusercontent.com/4140793/114928884-20f4c480-9e01-11eb-866f-f1df2541a2db.png)

**Comment by chalin** [comment]
@spzala - can we close this now that new docsy-themed site implements the changes suggested above?

**Comment by spzala** [comment]
@chalin sorry for the late reply, seems like I missed the GH notifications :( Thanks for working on the issue!

**Comment by chalin** [comment]
Thanks for giving a final ack.
